src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/privateApi/LicenceController.kt

```diff
@@ -1063,4 +1065,42 @@ class LicenceController(
   ) {
     updateSentenceDateService.updateSentenceDates(licenceId, request)
   }
+
+  @Tag(name = Tags.LICENCES)
+  @GetMapping(value = ["/getCutOffDate"])
```

Comment:
If we rename BankHolidayController to DatesController
and then have this as a GET, to `current_hard_stop_cutoff_date`